style/components/_navigation.scss

```diff
@@ -0,0 +1,12 @@
+.header {
+  margin-bottom: auto;
+  z-index: 2;
+}
+
+.nav__list {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  width: 80%;
```

Comment:
I think you probably shouldn't have a need to set the width if the whole area of the decor has padding, right?

Reply:
I removed the width and updated the padding values, but maybe we can discuss how it looks like and whether the design is reflected as  the designer might have think of it. ;p